DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -273,6 +273,12 @@ function AnyEntry()
     }
 
     ShowTeleporters();
+
+    if (dxr.flags.moresettings.empty_medbots > 0) {
```

Comment:
```suggestion
    if (dxr.flags.moresettings.empty_medbots > 0) { // this change isn't helpful until the next map reads from the defaults, but that's ok because we run this in the intro too
```